deposit/src/main/java/edu/unc/lib/deposit/work/DepositSupervisor.java

```diff
@@ -184,7 +185,8 @@ private void depositDuration(String depositUUID, Map<String, String> status) {
         long depositEndTime = System.currentTimeMillis();
         long depositTotalTime = depositEndTime - depositStartTime;
 
-        metricsClient.setDepositDuration(depositUUID, depositTotalTime);
+        GaugeFactory.createDurationGauge(MetricRegistry.name("deposit " + depositUUID, "deposit-duration"),
```

Comment:
might want to make it "deposit:" so that the uuid doesn't run into it.

Reply:
👍